crates/bevy_ecs/src/system/commands/mod.rs

```diff
@@ -205,6 +205,12 @@ impl<'w, 's> Commands<'w, 's> {
         }
     }
 
+    /// Returns a new [`Commands`] instance which uses the provided [`CommandQueue`],
+    /// reusing the [`Entities`] from the current [`Commands`] instance.
+    pub fn with_queue<'ns>(&self, queue: &'ns mut CommandQueue) -> Commands<'w, 'ns> {
+        Commands::new_from_entities(queue, self.entities)
+    }
```

Comment:
I wasn't a fan of this addition on my version of this PR, but I still have no idea what viable workaround there really is.

Reply:
I Think it's okay, as maybe there is some uses to this kind of approach with commands (As shown in this PR)
I can't think as this being a footgun at all. Is there some kind of problem with this method?